src/components/LicensePicker/style.css

```diff
@@ -0,0 +1,25 @@
+.spdx-picker {
+  font-family: 'Montserrat', sans-serif !important;
```

Comment:
What the style pluggability story for independent components? If this were used in a different site that did not use Montserrat, how would they change. Related, we are likely to change the font of the clearlydefined site. can this be inherited/supplied somehow by the outer context?